File: lab1main.cpp
#include "main.h"

int lines::numberSymbols() {
    int symbols = 0;
    for (char i : text) {
        symbols++;
    }
    return symbols;
}

std::string lines::textWithout5() {
    std::string result;
    for (char i : text) {
        if (i != '5') {
            result += i;
        }
    }
    return result;
}

int main() {
    lines object1;
    std::cin >> object1.text;
    std::cout << std::endl;
    std::cout << object1.numberSymbols() << std::endl;
    std::cout << object1.textWithout5() << std::endl;
}

File: lab1Main.cs
using System;

class Lines
{
    public string Text { get; set; }

    // Метод вычисления длины строки
    public int NumberSymbols()
    {
        int symbols = 0;
        foreach (char i in Text)
        {
            symbols++;
        }
        return symbols;
    }

    // Метод удаления символа '5' из строки
    public string TextWithout5()
    {
        string result = "";
        foreach (char i in Text)
        {
            if (i != '5')
            {
                result += i;
            }
        }
        return result;
    }
}

class Program
{
    static void Main()
    {
        Lines object1 = new Lines();
        object1.Text = Console.ReadLine();

        Console.WriteLine();
        Console.WriteLine(object1.NumberSymbols());
        Console.WriteLine(object1.TextWithout5());
    }
}